src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -132,6 +132,8 @@
                                </div>`,
                 });
 
+                tooltipNode.title = '';
```

Comment:
We were always using `[title]` selector to get all nodes for tooltips, both for parsing and hiding all tooltips. So far Bootstrap was using `.setAttribute('title', '')` in its constructor, but in 5.2.0 they introduced `.removeAttribute('title')` instead, which removed `title` parameter, I set it back here, so that I could still use same selector for getting all tooltips

Reply:
Really appreciate your explanation! 😊

Reply:
If you `parse` a second time the same tooltipNode and title is the empty string will bootstrap change the tooltip to empty string?

Reply:
not anymore, after last commit ;)